x-pack/plugin/ilm/qa/multi-cluster/build.gradle

```diff
@@ -8,13 +9,20 @@ dependencies {
     testCompile project(':x-pack:plugin:ilm')
 }
 
-task leaderClusterTest(type: RestIntegTestTask) {
+File repoDir = file("$buildDir/testclusters/repo")
+
+task 'leader-cluster'(type: RestIntegTestTask) {
     mustRunAfter(precommit)
+    runner {
+        systemProperty 'tests.target_cluster', 'leader'
+        /* To support taking index snapshots, we have to set path.repo setting */
+        systemProperty 'tests.path.repo', repoDir.absolutePath
+    }
 }
 
-leaderClusterTestCluster {
-    numNodes = 1
-    clusterName = 'leader-cluster'
+testClusters.'leader-cluster' {
+    distribution = "DEFAULT"
+    setting 'path.repo', repoDir.absolutePath
```

Comment:
I think it's configured in a single place for multiple projects. I wanted to make it explicit on the cluster configuration for awareness since we are planning to discourage it.